modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -996,6 +996,34 @@ CV__DNN_EXPERIMENTAL_NS_BEGIN
                              CV_OUT std::vector<int>& indices,
                              const float eta = 1.f, const int top_k = 0);
 
+     class CV_EXPORTS_W_SIMPLE Model : public Net
+     {
+     public:
+         Model(const std::string& model, const std::string& config = "", int width = -1,
+               int height = -1, Scalar mean = Scalar(), float scale = 1.0,
+               bool swapRB = true, bool crop = false);
+
+         void setInputWidth(int width);
+         void setInputHeight(int height);
```

Comment:
Don't split width and height. It is a bad practice. Use `setInputSize(const Size& sz)` instead.

Reply:
I think we can also have something like void `setInputSize(int width, int height)`. Just more friendly for Python:

```python
model.setInputSize(width=300, height=300)
```
then
```python
model.setInputSize((300, 300))
```